Utilities/Log.cpp

```diff
@@ -1,4 +1,4 @@
-﻿#include "Log.h"
+#include "Log.h"
```

Comment:
What did you do there again?

Reply:
No idea , I assume something must be wrong with my github but I have no idea what , will reinstall.